pom.xml

```diff
@@ -0,0 +1,417 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://maven.apache.org/POM/4.0.0"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>com.exasol</groupId>
+    <artifactId>maven-plugin-integration-testing</artifactId>
+    <version>0.1.0</version>
+    <name>maven-plugin-integration-testing</name>
```

Comment:
I think name is usually a human-readable variant of artifactId, not the complete copy